### PR TITLE
Add prisma-docs-search extension

### PR DESCRIPTION
**Pull request by petradonka**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — COMMENTED** [review]
Where does this extension get its data from? Are you hosting the [endpoint](https://ask-ai-proxy.raycast-0ef.workers.dev/query/v1/projects/6af273b4-8b7b-4569-b76d-0ba12ccb88bc/chat/stream/)? 😊

File: extensions/prisma-docs-search/package.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  "access": "public",
+  "keywords": [
+    "Prisma",
+    "Search",
+    "AI Search"
+  ],
+  "commands": [
+    {
+      "name": "prisma-docs-search",
+      "title": "Search",
```
Comment (pernielsentikaer) [inline extensions/prisma-docs-search/package.json:27]:
```suggestion
      "title": "Search Prisma Docs",
```

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by petradonka** [comment]
Thanks for the help with this PR @pernielsentikaer! 🙏 

We've shared the news of the extension from our Prisma accounts, Raycast makes some cool stuff possible :) 

https://x.com/prisma/status/1842193695158022604

**Pull request merged**